packages/klighd-core/src/proxy-view/proxy-view-cluster.tsx

```diff
@@ -0,0 +1,150 @@
+import { VNode } from "snabbdom";
+
+/** Returns the rendering of clusters. */
+export function getClusterRendering(id: string, numProxies: number, size: number, x: number, y: number): VNode {
+    const squareSize = size * 0.9;
+    const squareOffset = size * 0.1;
+    const textSize = size * 0.5;
+    return JSON.parse(
```

Comment:
I still do not like this very much. What attribute prevented you from simply using jsx here?

Reply:
`width` and `height` only have getters and couldn't be set easily. This seemed like the simplest way to get around this

Reply:
Compare to the `renderSingleSVGRect` method in views-rendering: There the width and height are just set as attributes using jsx, so that alone cannot be the reason